dali/python/nvidia/dali/pipeline.py

```diff
@@ -518,6 +518,11 @@ def feed_input(self, data_node, data, layout="", cuda_stream = None):
             If internal stream is used, the call to ``feed_input`` will block until the copy to
             internal buffer is complete, since there's no way to synchronize with this stream to
             prevent overwriting the array with new data in another stream.
+
+        `no_copy` : If DALI should copy the buffer then feed_input is called
+            If set to true it is the user responsibility to keep buffer alive and unmodified
+            until it is used in the pipeline. The rule of thumb is to keep alive 3
+            (number of DALI stages) * prefetch_queue_depth
```

Comment:
```suggestion
        `no_copy` : If DALI should copy the buffer when feed_input is called
            If ``no_copy`` is set to true instead of making a copy of the provided buffer, 
            DALI passes the user's memory directly in the Pipeline.
            It is user's responsibility to keep the buffer alive and unmodified
            until it is used in the pipeline. 
            
            The buffer can be modified again after the outputs of the iteration it was used in were consumed, which can happen `prefetch_queue_depth` iterations after the `feed_input` call.
```

The keeping alive 3 * prefetch_queue_depth is a bit enigmatic.

Reply:
Mhhh, that is right

Reply:
Done